login-workflow/src/contexts/RegistrationContext/types.ts

```diff
@@ -11,12 +11,54 @@ export type AccountDetails = {
 
 // this should be updated to add new actions for greater control
 export type RegistrationUIActions = {
+    /**
+     * A function that is used to load the EULA.
+     * @param {string} language - This function will be called when the user lands on the EULA screen
+     * @returns Promise<string>
+     */
     loadEula?: (language: string) => Promise<string>;
+
+    /**
+     * A function that is used to accept the EULA. This function will be called when the user clicks the Next button on the EULA screen
+     * @returns Promise<void>
+     */
     acceptEula?: () => Promise<void>;
+
+    /**
+     * A function that is used to request a registration code.
+     * @param {string} email - This function will be called when the user lands on the Verify Code screen as well as when a user clicks the Resend Verification Code button
+     * @returns Promise<string>
+     */
     requestRegistrationCode?: (email: string) => Promise<string>;
+
+    /**
+     * A function that is used to validate a registration code. This function will be called when the user clicks the Next button on the Verify Code screen screen
+     * @param {string} validationCode - verify code input
+     * @param {string} validationEmail - verify email input
+     * @returns Promise<boolean>
+     */
     validateUserRegistrationRequest?: (validationCode: string, validationEmail?: string) => Promise<boolean>;
+
+    /**
+     * A function that is used to create a password. This function will be called when the user clicks the Next button on the Create Password screen
+     * @param {string} password - validates password input
+     * @returns Promise<boolean>
+     */
     createPassword?: (password: string) => Promise<boolean>;
+
+    /**
+     * A function that is used to set the account details. This function will be called when the user clicks the Next button on the Account Details screen
+     * @param {AccountDetails} details - verify first and last name input
+     * @returns Promise<boolean>
+     */
     setAccountDetails?: (details: AccountDetails) => Promise<boolean>;
+
+    /**
+     * A function that is used to complete the registration. This function will be called when the user clicks the Next button on the final registration workflow screen
+     * @param {string} email - Used to display on final screen
+     * @param {string} organizationName - Used to display on final screen
```

Comment:
These are not the params here...the params are userData and validationCode. email and organizationName are part of the @returns value...if you want to describe what they are used for you can do it at the end of the @returns line